kgo.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const kgo = require('kgo');
+const shuv = require('shuv');
+const foreign = require('foreign');
```

Comment:
:abc: Alphabetize the third-party imports, and follow the imports with two empty lines, please. :)